invoices_app/views.py

```diff
@@ -444,7 +448,7 @@ def post_a_comment(request, pk):
         message=request.POST['message'],
         comment_file=request.FILES.get('invoice_file'),
     )
-
+    Invoice.objects.filter(pk=invoice.id).update(new_comment_from_supplier=True)
```

Comment:
Same here use get instead of filter or use invoice instance invoice.new_comment_from_supplier = True and then invoice.save()